oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/RdbConnectionUtils.java

```diff
@@ -148,4 +155,22 @@ private static boolean checkImageAvailability() throws TimeoutException {
     private static boolean checkDockerAvailability() {
         return DockerClientFactory.instance().isDockerAvailable();
     }
+
+    private static Map<String, String> parseDockerEnv(String raw) {
+        if (StringUtils.isEmpty(raw)) {
```

Comment:
do we need to special-case this? Won't the tokenizer just return no tokens on an empty string?

Reply:
The only difference in behavior would be that we would get a warning on something like -Drdb.docker-env='     ', but that's an improvement. I'll change it.